### PR TITLE
DDP-6563 fix activity steps styles

### PR DESCRIPTION
**Pull request by aglushko-broad**
mockups https://projects.invisionapp.com/d/main#/console/21368369/455066475/preview
before
image

after
image

### CODE REVIEW

**Comment by alextanlan** [comment]
in Osteo and Brain studies the `Consent` activity steps looks differently.
I guess Pancan should look similar if we don't have special requirements, do we? 
May be it should be discussed with Erin / Marco ?

screen1

screen2

File: ddp-workspace/projects/ddp-pancan/src/styles/activities.scss [hunk 1]
```diff
@@ -82,34 +82,70 @@
...
+        padding-left: 8px;
+    }
 
     &.completed {
         cursor: pointer;
-        color: $activity-step-completed;
 
+        .activity-step {
+            &__number {
+                background-color: #5a42d0;
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-pancan/src/styles/activities.scss]: `$activity-step-inactive` can be used here.

**Comment by aglushko-broad** [comment]
> in Osteo and Brain studies the `Consent` activity steps looks differently.
> I guess Pancan should look similar if we don't have special requirements, do we?
> May be it should be discussed with Erin / Marco ?
> 
> screen1
> 
> screen2

Mocups look like a special requirement to me https://projects.invisionapp.com/d/main#/console/21368369/455066475/preview. So we have requirements. It was discussed and confirmed with Marco

File: ddp-workspace/projects/ddp-pancan/src/styles/activities.scss (hunk 1 above)
Comment (aglushko-broad) [reply ddp-workspace/projects/ddp-pancan/src/styles/activities.scss]: good catch! done

**Review by alextanlan — APPROVED** [review]

**Pull request merged**